src/components/Sidebar/Sidebar.js

```diff
@@ -42,126 +50,175 @@ const useStyles = makeStyles((theme) => ({
-                    >
+                  </ListItem>
+                  <ListItem button key='Facilities'
+                    onClick={() => {props.handleOpen(); history.push('/facilities')}}
+                    style={(currentLocation === '/facilities') ? (active) : (null)}
+                  >
+                    <ListItemIcon><img src={FacilitiesIcon} alt="facilities icon" /></ListItemIcon>
+                    <ListItemText>{t('Facilities')}</ListItemText>
+                  </ListItem>
+                  <ListItem button key='Profile' onClick={() => {props.handleOpen(); history.push('/profile')}}
+                    style={(currentLocation === '/profile') ? (active) : (null)}
+                  >
                     <ListItemIcon><img src={RolesIcon} alt="roles icon" /></ListItemIcon>
-                    <ListItemText>Roles</ListItemText>
-                    </ListItem> 
-                    <ListItem button key='AddUser' onClick={() =>{ props.handleOpen()}}
-                     style={(currentLocation === '/addUser') ? (active) : (null)}
-                    >
+                    <ListItemText>Profile</ListItemText>
+                  </ListItem> 
+                  <ListItem button key='AddUser' onClick={() =>{ props.handleOpen()}}
+                    style={(currentLocation === '/addUser') ? (active) : (null)}
+                  >
                     <ListItemIcon><img src={AddUserIcon} alt="add user icon" /></ListItemIcon>
                     <ListItemText>Add User</ListItemText>
-                    </ListItem>
-                    <ListItem 
-                    onClick={() => {props.handleOpen(); history.push('/facilities')}}
-                    button key='Facilities'
+                  </ListItem>
+                  <ListItem button key='Roles' onClick={() => {props.handleOpen()}}
                     style={(currentLocation === '/userrole') ? (active) : (null)}
-                    >
+                  >
+                    <ListItemIcon><img src={RolesIcon} alt="roles icon" /></ListItemIcon>
+                    <ListItemText>Roles</ListItemText>
+                  </ListItem> 
+                  <ListItem button key='Hotels' onClick={() => {props.handleOpen(); history.push('/hotel')}}
+                    style={(currentLocation === '/hotel') ? (active) : (null)}
+                  >
+                    <ListItemIcon><HomeWorkIcon /></ListItemIcon>
+                    <ListItemText>Hotels</ListItemText>
+                  </ListItem> 
+                  <ListItem button key='FAQ' onClick={() => {props.handleOpen(); history.push('/deletefaq')}}
+                    style={(currentLocation === '/deletefaq') ? (active) : (null)}
+                  >
+                    <ListItemIcon><NotListedLocationIcon /></ListItemIcon>
+                    <ListItemText>FAQ</ListItemText>
+                  </ListItem> 
+                </>
+              ) : (userRole === 2) 
+              ? (
```

Comment:
How about other managers? since you are checking the `roleId ===2` and forex `Travel manger` has role id of 3. Do you think will be have anything on the sidebar with this implementation?
Please consider that.